code/game/objects/items/explosives/plastic.dm

```diff
@@ -195,7 +195,7 @@
 	if(istype(target, /turf/closed/wall))
 		var/turf/closed/wall/W = target
 		if(W.hull)
-			to_chat(user, SPAN_WARNING("You are unable to stick \the [src] to the [W]!"))
+			to_chat(user, SPAN_WARNING("You are unable to stick \the [src] to [W]!"))
```

Comment:
If you're opting not to use `\the` and instead redact it - then should be consistent and remove it here.